src/components/telemetry_monitor/include/telemetry_monitor/metric_wrapper.h

```diff
@@ -51,7 +51,7 @@ class MetricWrapper {
   virtual std::string GetStyledString();
   virtual Json::Value GetJsonMetric();
   virtual ~MetricWrapper();
-
+  virtual const utils::ResourseUsage& getResources();
```

Comment:
@BSolonenko 
- named wrongly according to SDL Coding Style

Reply:
@AByzhynar fixed in https://github.com/Ford-Luxoft/sdl_core/pull/30/commits/5e15fe123252c49dde19d2d32cd757f1ac7eacf8